usecases/schema/tenant.go

```diff
@@ -122,6 +132,29 @@ func validateTenants(tenants []string) error {
 	return nil
 }
 
+func validateActivityStatuses(statuses []string, allowEmpty bool) error {
```

Comment:
Why do we `allowEmpty` for tenant updates?

Reply:
actually we don't. Method is used for validation input of create and update methods. Empty is allowed for creation (`true`; `hot` is then used as default) and required for update (`false`).

Reply:
I had noticed that the update tenants code path calls `validateActivityStatuses(statuses, false)`, so was wondering what is the reason, but I see I got it the other way around, that we allow empty status for creating tenants.

But now I see the default `HOT` status:
```
func ActivityStatus(status string) string {
	if status == "" {
		return models.TenantActivityStatusHOT
	}
	return status
}
```